avutil/avutil.go

```diff
@@ -980,6 +989,64 @@ func (f *Frame) PacketDuration() int64 {
 	return int64(C.av_frame_get_pkt_duration(f.CAVFrame))
 }
 
+func (f *Frame) GetSideData() *FrameSideData {
+	cSideData := C.av_frame_get_side_data(f.CAVFrame, C.AV_FRAME_DATA_MOTION_VECTORS)
+	if cSideData == nil {
+		return nil
+	}
+
+	return &FrameSideData{
+		CFrameSideData: (*C.AVFrameSideData)(unsafe.Pointer(cSideData)),
+	}
+}
+
+type FrameSideData struct {
+	CFrameSideData *C.AVFrameSideData
+}
+
+func (sd *FrameSideData) Size() int {
+	return int(C.go_motion_vector_size(sd.CFrameSideData))
+}
+
+func (sd *FrameSideData) MotionVector(n int) *MotionVector {
+	mv := C.go_motion_vector(sd.CFrameSideData, C.int(n))
+	return &MotionVector{
+		CMotionVector: mv,
+	}
+}
+
+type MotionVector struct {
+	CMotionVector *C.AVMotionVector
+}
+
+func (mv *MotionVector) Source() int64 {
+	return int64(mv.CMotionVector.source)
+}
+
+func (mv *MotionVector) Width() int64 {
+	return int64(mv.CMotionVector.w)
+}
+
+func (mv *MotionVector) Height() int64 {
+	return int64(mv.CMotionVector.h)
+}
+
+func (mv *MotionVector) SrcPosition() (int64, int64) {
```

Comment:
Any reason why you use `int64`?
libav returns `int16` for this: https://www.ffmpeg.org/doxygen/3.0/structAVMotionVector.html#af1cf3b7f185df7d95107625bc10c5c59
I think we should use `int16` too. Same for `DstPosition`.